test/validators/locations/create.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const Joi            = require('joi');
```

Comment:
Nit: You don't actually have to align the Joi require - we usually separate 3rd party requires from internal module requires with a newline. Aligning `=`'s is done separately between those two groups (3rd party/internal modules)